features.yaml

```diff
@@ -29,6 +29,12 @@ features:
       maturity: Stable
       nextExpectedPromotion: ""
     area: Traffic Management
+  - name: "Gateway Injection"
+    level:
+      checklist: ""
+      maturity: Stable
```

Comment:
Fix maturity. I assume this is either alpha or experimenta?

Reply:
Oops, thanks. It want to go to alpha ideally, but can put experimental if we are not ready. will update

Reply:
Since external control plane is alpha and is using it, it would be best to make this alpha as well. Otherwise, wouldn't we need to demote external control plane?

Reply:
Let's try for alpha, put it on agenda for next TOC to get the TOC approval.

Reply:
What's the level of testing we have done for this feature @howardjohn ?

Injection templates have been flaky and used for several things, I think we should start with experimental here unless we have higher confidence of this working.

Reply:
We have e2e tests that ensure we can inject a gateway through various installation methods (helm and plain yaml) and that traffic works, etc. We also have extensive unit tests of injection.

I am not sure what you mean by injection templates have been flaky - they are one of the oldest features of Istio.

Additionally, this is the only mechanism to install gateways for external control plane, so we have a high level of testing of this feature by end users.

The alpha bar:
"The feature is covered by unit tests and integration tests where the feature is enabled and the tests are non-flaky. Tests may not cover all corner cases, but the most common cases have been covered." which I feel we have met.